### PR TITLE
Remove non-react props from the rendered component

### PR DESCRIPTION
**Pull request by doylem**

### CODE REVIEW

**Comment by thoiberg** [comment]
Do we need to bump the version? I'm thinking there may be problems trying to install this if it matches a version of a previous install (yarn caching)

**Comment by doylem** [comment]
@thoiberg yeah I usually tag a new release (which bumps the version number) on master once the PR is merged.

File: dist/react-selectable-extended.js [hunk 1]
```diff
@@ -108,6 +108,8 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
+	function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }
```
Comment (vaevictis) [inline dist/react-selectable-extended.js:111]: ~~Isn't that a little bit of a brutal one-liner?~~
Ah, `dist` folder...

**Review by thoiberg — APPROVED** [review]
LGTM

**Review by vaevictis — APPROVED** [review]
Good riddance

File: dist/react-selectable-extended.js (hunk 1 above)
Comment (doylem) [reply dist/react-selectable-extended.js:111]: yeah, TBH i'm not even sure why we commit the dist/ stuff - but i'm just following what the original plugin did :shrug:

**Pull request merged**